### PR TITLE
switched back to .url

### LINKED ISSUE #85: Main navigation is broken
It looks like this pull request https://github.com/StefMa/hugo-fresh/pull/78/files broke the main navigation. As far as I understand the idea was to fix hugo warnings by replacing .URL with .Permalink in the templates, but only one of those replacements was needed. The others were attributes in the config file.

### PR DESCRIPTION
**Pull request by nathanbiller**
Fixes #85

### CODE REVIEW

File: layouts/partials/single/sidebar.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <ul class="sidebar-menu">
       {{ range where .Site.RegularPages "Section" (strings.TrimSuffix "/" .File.Dir) }}
       <li>
-        <a href="{{ .Permalink }}">
+        <a href="{{ .url }}">
```
Comment (todinov) [inline layouts/partials/single/sidebar.html]: Here it should be .Permalink, because we are iterating over Page objects.

**Comment by StefMa** [comment]
@todinov could you please check if everything is correct now? Then I would merge it ☺

**Comment by todinov** [comment]
> @todinov could you please check if everything is correct now? Then I would merge it ☺

Just tested it and everything seems to work correctly.

**Review by todinov — APPROVED** [review]

**Comment by StefMa** [comment]
Thank you

**Pull request merged**